src/composables/trade/useValidation.ts

```diff
@@ -12,7 +23,7 @@ export default function useValidation(
   const auth = useAuth();
 
   const validationStatus = computed(() => {
-    if (!auth.isAuthenticated) return '';
+    if (!auth.isAuthenticated.value) return TradeValidation.NO_ACCOUNT;
```

Comment:
Standartizing "not authenticated" error + fixed a bug where the condition was always false.